Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -144,6 +162,32 @@ protected override bool ControlPropertiesValid ()
         return NamingContainer.FindControl(controlToValidate) != null;
     }
 
+    private void DispatchEditableRowValidationFailures (
+        IEditableRow editableRow,
+        IBusinessObject rowObject,
+        IReadOnlyList<BocColumnDefinition> columnDefinitions,
+        IBocListValidationFailureRepository validationFailureRepository)
+    {
+      var castedEditableRow = (EditableRow)editableRow;
```

Comment:
We can move the relevant APIs to the interface. Perhaps not just GetValidators but also the related HasValidators etc methods?

Reply:
`EditableRow` has a lot of members that are not part of the interface. Right here we only need the `GetValidators` method. If you want to include other members as well we should talk about what members you want to move.

Reply:
For symmerty with GetEditControls:
* HasValidators()
* HasValidator(int)
* GetValidators(int)

that's it for now.